LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
do we need both `LICENSE` and `LICENSE.md` files?

Reply:
I can remove LICENSE.md, since it was already present I thought not to touch it.

For PyPI we need LICENSE. 

Let me update the PR and remove it (LICENSE.md)